### DISCUSSION TITLE
J101 Enable SD Card | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#121] [Q&A]
# J101 Enable SD Card | Seeed Studio Wiki

J101 Enable SD Card

https://wiki.seeedstudio.com/J101_Enable_SD_Card/

**Comment by oguzozcann** [comment]
/boot$ sudo /opt/nvidia/jetson-io/config-by-hardware.py -l
Help Please...
lsblk: unknown column: mountpoint,partlabel
Traceback (most recent call last):
  File "/opt/nvidia/jetson-io/config-by-hardware.py", line 125, in <module>
    main()
  File "/opt/nvidia/jetson-io/config-by-hardware.py", line 94, in main
    jetson = board.Board()
  File "/opt/nvidia/jetson-io/Jetson/board.py", line 207, in __init__
    mountpoint = _board_partition_is_mounted('APP')
  File "/opt/nvidia/jetson-io/Jetson/board.py", line 131, in _board_partition_is_mounted
    entries = syscall.call_out('lsblk -n -r -o mountpoint,partlabel')
  File "/opt/nvidia/jetson-io/Utils/syscall.py", line 33, in call_out
    output = subprocess.check_output(cmd, shell=True)
  File "/usr/lib/python3.6/subprocess.py", line 356, in check_output
    **kwargs).stdout
  File "/usr/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command 'lsblk -n -r -o mountpoint,partlabel' returned non-zero exit status 1.

**Comment by MatthewJeffson** [comment]
Welcome to our product documentation platform! We value user feedback and contributions, so we couldn’t be more grateful that you can help us improving the documentation.

Hence, if you find obvious errors in the product documentation, or you have additional ideas and insights about using products, please [contribute](https://wiki.seeedstudio.com/Contribution-Guide) to us.

If you need technical support, you can email it to [fae@seeed.cc](mailto:fae@seeed.cc).

If you want to discuss the product, I recommend the [Seeed Studio Forum](https://forum.seeedstudio.com/) and [Seeed Studio Discord Channel](https://discord.gg/eWkprNDMU7)

The "Discussion channel" in Github here is a relatively new addition and may not receive much attention.

Nonetheless, we hope you find it helpful with the help of others. We will clean it up on a monthly basis.
